pyproject.toml

```diff
@@ -28,6 +28,7 @@ extend-ignore = [
     "D409",
     "D413",
     "D107",
+    "E501",
```

Comment:
I think we recently removed this, so this might be controversial, so I'll point it out.

I think the best approach here is to let `ruff format` deal with long lines, as it won't try to break up strings and it doesn't require us to put `#  noqa` by every test with a long name.